File: app/src/main/java/ru/ulstu_team/ulstuschedule/ui/faculties/FacultiesMvpView.java
package ru.ulstu_team.ulstuschedule.ui.faculties;

import java.util.List;

import ru.ulstu_team.ulstuschedule.data.model.Faculty;
import ru.ulstu_team.ulstuschedule.ui.base.MvpView;

public interface FacultiesMvpView extends MvpView {

    void showFaculties(List<Faculty> faculties);

    void showEmptyList();

    void showError();

}

File: app/src/main/java/ru/ulstu_team/ulstuschedule/ui/old/faculties/FacultiesMvpView.kt
package ru.ulstu_team.ulstuschedule.ui.old.faculties

import ru.ulstu_team.ulstuschedule.data.model.Faculty
import ru.ulstu_team.ulstuschedule.ui.base.MvpView

interface FacultiesMvpView : MvpView {

    fun showFaculties(faculties: List<Faculty>)

    fun showEmptyList()

    fun showError()

}
